src/trigger.ts

```diff
@@ -77,3 +78,74 @@ export async function removeMapMark(id: number): Promise<void> {
     )
   )
 }
+
+export async function smoke(
+  position: PositionLL,
+  color : number
```

Comment:
This needs to either be a `Color` Enum or a string, number here is not useful to anyone (enums explicitly solve the mapping a number to a meaning problem)

Reply:
changed as requested in https://github.com/bitbubbler/yet-another-dcs-server/pull/13/commits/c739f19620b4bd3fa549c0263fe09e99040170f6

using the color enums from DCS now to call smoke and flare functions